package-drone/src/main/java/de/dentrassi/pm/jenkins/UploaderV2.java

```diff
@@ -93,7 +91,7 @@ private URI makeUrl ( final String file ) throws URIException, IOException
     }
 
     @Override
-    public void upload ( final File file, final String filename ) throws IOException
+    public void addArtifact ( final File file, final String filename ) throws IOException
```

Comment:
Collect artifacts to the internal map and than performs the upload in the performUpload method